cpp/data/system_updates/7.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -o errexit -o nounset
+
+readonly SYSTEMD_SERVICE_DIR=/etc/systemd/system/
+mkdir --parents $(SYSTEMD_SERVICE_DIR)
```

Comment:
i think mkdir should be removed.... if the directory doesnt exist, there will be no systemd installation and copying the file there will not have any effect